ftplugin/ledger.vim

```diff
@@ -347,9 +404,41 @@ function! s:count_expression(text, expression) "{{{2
   return len(split(a:text, a:expression, 1))-1
 endf "}}}
 
+function! s:autocomplete_account_or_payee(argLead, cmdLine, cursorPos) "{{{2
+  return (a:argLead =~ '^@') ?
+        \ map(filter(systemlist(g:ledger_bin . ' -f ' . shellescape(expand('%')) . ' payees'),
+        \ "v:val =~? '" . strpart(a:argLead, 1) . "'"), '"@" . v:val')
+        \ :
+        \ filter(systemlist(g:ledger_bin . ' -f ' . shellescape(expand('%')) . ' accounts'),
+        \ "v:val =~? '" . a:argLead . "'")
+endf "}}}
+
+function! s:reconcile(account) "{{{2
+  " call inputsave()
+  let l:amount = input('Target amount' . (empty(g:ledger_default_commodity) ? ': ' : ' (' . g:ledger_default_commodity . '): '))
+  " call inputrestore()
+  call ledger#reconcile(a:account, l:amount)
+endf "}}}
+
 " Commands {{{1
```

Comment:
Maybe we should use `:command!` to overwrite existing definitions (and remove the `if` condition) while using `-buffer` to restrict the commands to the ledger buffer (see `:help :command-buffer`)?